coderd/database/queries/templates.sql

```diff
@@ -120,9 +120,10 @@ SET
 	max_ttl = $6,
 	autostop_requirement_days_of_week = $7,
 	autostop_requirement_weeks = $8,
-	failure_ttl = $9,
-	time_til_dormant = $10,
-	time_til_dormant_autodelete = $11
+	autostart_block_days_of_week = $9,
```

Comment:
Storing a blocklist instead of an allowlist seems strange to me.

Reply:
Yea, however I really wanted the zero value to be the correct default value.

If I flip this, then we have to store `0b01111111` or `127`, which feels very "magical" from a default POV.

Reply:
That makes sense 👍